### ISSUE TITLE
Substitute combining characters with available glyphs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#295]
```
Truetype fonts typically do not contain glyphs for combining characters.
This makes using them quite useless:)
If it is feasible to overprint non-combining glyphs in one display position, 
I'd suggest the following:

For common combining characters that have corresponding Unicode non-combining 
glyphs, the latter could be used as substitutes (if the respective real 
combining is not available in the font); specifically:

̀ U+0300;COMBINING GRAVE ACCENT
` U+0060;GRAVE ACCENT

́ U+0301;COMBINING ACUTE ACCENT
´ U+00B4;ACUTE ACCENT

 ̂ U+0302;COMBINING CIRCUMFLEX ACCENT
(^ U+005E;CIRCUMFLEX ACCENT - too big)
ˆ U+02C6;MODIFIER LETTER CIRCUMFLEX ACCENT

̃ U+0303;COMBINING TILDE
(~ U+007E;TILDE - often too big)
˜ U+02DC;SMALL TILDE

̄ U+0304;COMBINING MACRON
(¯ U+00AF;MACRON - too big?)
ˉ U+02C9;MODIFIER LETTER MACRON

̆ U+0306;COMBINING BREVE
˘ U+02D8;BREVE

̇ U+0307;COMBINING DOT ABOVE
˙ U+02D9;DOT ABOVE

̈ U+0308;COMBINING DIAERESIS
¨ U+00A8;DIAERESIS

 ̊ U+030A;COMBINING RING ABOVE
˚ U+02DA;RING ABOVE

̋ U+030B;COMBINING DOUBLE ACUTE ACCENT
˝ U+02DD;DOUBLE ACUTE ACCENT

 ̌ U+030C;COMBINING CARON
ˇ U+02C7;CARON

̧ U+0327;COMBINING CEDILLA
¸ U+00B8;CEDILLA

̨ U+0328;COMBINING OGONEK
˛ U+02DB;OGONEK

```

Original issue reported on code.google.com by `towom...@googlemail.com` on 22 Sep 2011 at 11:08

**Comment by mintty** [comment]
The proposed substitution is actually not needed after tweaking the display of combining characters.
See also #553.

**Comment by mintty** [comment]
Released combining character display fix with 2.4.0.
